packages/cells/src/widget.ts

```diff
@@ -690,6 +690,13 @@ class MarkdownCell extends Cell {
   readonly model: IMarkdownCellModel;
 
   /**
+   * A promise that resolves when the widget renders for the first time.
+   */
+  get ready(): Promise<void> {
```

Comment:
Other parts of the code base have a ready promise that only fires the first time it is ready.  This avoids having to have a signal for each of these things, and is only there because initialization is asynchronous.